# Description: Since a constant is passed into strchr(), pu must also be const
#              to keep the compiler from generating an error.
# Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/gbgoffice/+bug/445624
# Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=550115
#
--- a/src/translator.cpp
+++ b/src/translator.cpp
@@ -315,7 +315,8 @@ bool Translator::findWord(const char *wo
 
 char *Translator::transformResult(const char *result) {
 	int j, i, m, n, u;
-	char *b, *pu;
+	const char *pu;
+	char *b;
 	char c;
 	strcpy(dataBuffer2, result);
 	if ((advancedSearchState) && (advancedSearchHighlight)) {
